runtimevar/driver/driver.go

```diff
@@ -39,20 +39,33 @@ type Variable struct {
 // dictate the type of Variable.Value and a decoding function.  The Watcher provider can use the
 // runtimevar.Decoder to facilitate the decoding logic.
 type Watcher interface {
-	// WatchVariable blocks until the variable changes, the Context's Done channel closes or an
-	// error occurs.
+	// WatchVariable returns one of:
```

Comment:
I re-read this, and I don't have any concrete suggestions on how to improve clarity in this iteration. Let's focus on this when we have the interface we like.